chainConfigs/5.json

```diff
@@ -0,0 +1,5 @@
+{
+  "FACTORY_OWNER": "0xDC498668B5e6CC518fD58A2ADBF614Fd3A13D3a0",
```

Comment:
this value is new - where does it come from before?

Reply:
It's querying the factory to get the owner